source/sass/views/youtube-regrets.scss

```diff
@@ -3,6 +3,170 @@
     background: linear-gradient($youtube-regrets-light-red, $white);
   }
 
+  .intro-viewport {
+    --scenePerspective: 50;
+    --blockZTranslate: 0;
+    --ringZTranslate: 0;
+    --sceneDepth: 0;
+
+    $numRings: 5;
```

Comment:
I can't find a way to not use hardcoded value here

Reply:
Cool. Never thought about using counter. I will give it a try!

Reply:
Tried it and unfortunately, it didn't work. I have replaced the SCSS for loop with 

```scss
&:nth-child(1) {
  transform: translate3D(0, 0, get-ring-z-translate(1));
}

&:nth-child(2) {
  transform: translate3D(0, 0, get-ring-z-translate(2));
}

&:nth-child(3) {
  transform: translate3D(0, 0, get-ring-z-translate(3));
}

&:nth-child(4) {
  transform: translate3D(0, 0, get-ring-z-translate(4));
}

&:nth-child(5) {
  transform: translate3D(0, 0, get-ring-z-translate(5));
}
```

It looks repetitive and redudent but this is more explicitly and we are already doing the same for `.block` so :see_no_evil: 🤷‍♀

Reply:
Let's leave it as is, then, and maybe revisit it if and when we care to.